astropy/modeling/fitting.py

```diff
@@ -840,6 +838,61 @@ def _convert_input(x, y, z=None, n_models=1, model_set_axis=0):
     return farg
 
 
+class MinimizeFitter(Fitter):
+    """
+
+    A fitter for the minimize optimizer.
+
+    Parameters
+    ----------
+    method : str or callable(custom)
+        select minimization method
+
+    supported_constraints : list
+        lists the constraint types supported by the Optimization method default is fixed
```

Comment:
Just read over this again. There is also a "." missing after "method" I believe.